### ISSUE TITLE
cmd/tsuru: plan-list is broken at master

### ISSUE THREAD

**Issue by fsouza** [tsuru/tsuru#1133]
```
% tsuru plan-list
plan-List flag redefined: h
panic: flag redefinition

goroutine 1 [running]:
launchpad.net/gnuflag.(*FlagSet).Var(0xc20806a2d0, 0x8719a8, 0xc20809f2f0, 0x518ab0, 0x1, 0x54ffb0, 0x15)
	/Users/f/src/launchpad.net/gnuflag/flag.go:653 +0x3c1
launchpad.net/gnuflag.(*FlagSet).BoolVar(0xc20806a2d0, 0xc20809f2f0, 0x518ab0, 0x1, 0xc20809f200, 0x54ffb0, 0x15)
	/Users/f/src/launchpad.net/gnuflag/flag.go:436 +0x8e
github.com/tsuru/tsuru/cmd.(*Manager).handleFlags(0xc208078d20, 0x871540, 0xc20809f1a0, 0x7fff5fbff03e, 0x9, 0xc20809f2a0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/f/src/github.com/tsuru/tsuru/cmd/cmd.go:230 +0x1f5
github.com/tsuru/tsuru/cmd.(*Manager).Run(0xc208078d20, 0xc20809f2a0, 0x0, 0x0)
	/Users/f/src/github.com/tsuru/tsuru/cmd/cmd.go:173 +0x17ba
main.main()
	/Users/f/src/github.com/tsuru/tsuru-client/tsuru/main.go:85 +0xc5

goroutine 4 [runnable]:
os/signal.loop()
	/Users/f/opt/src/go/src/os/signal/signal_unix.go:20
created by os/signal.init.1
	/Users/f/opt/src/go/src/os/signal/signal_unix.go:28 +0x3f

goroutine 6 [runnable]:
github.com/tsuru/tsuru/db/storage.retire(0xc20803c440)
	/Users/f/src/github.com/tsuru/tsuru/db/storage/storage.go:106
created by github.com/tsuru/tsuru/db/storage.init.1
	/Users/f/src/github.com/tsuru/tsuru/db/storage/storage.go:102 +0x6c
```

Renaming the flag would do the job. We could also change the behavior and have human readable sizes as the default value, and have the -r/--robot mode.

We could also switch to the Linux ``free`` behavior, having memory displayed in kilo, mega or gigabytes.

All proposed solutions are easy to apply, so I'm marking this as easy.

**Comment by fsouza** [comment]
This isn't only at master, as this change has been released in [tsuru 0.15.1](https://github.com/tsuru/tsuru-client/releases/tag/0.15.1).

**Comment by fsouza** [comment]
After some discussion, we decided to keep the ``-h`` and fix the panic itself, so we don't register the ``-h`` nor ``--help`` when these flags are already present.
